app-misc/mx5000tools/mx5000tools-0.1.2.ebuild

```diff
@@ -13,8 +13,12 @@ LICENSE="GPL-2+"
 SLOT="0"
 KEYWORDS="amd64 ppc ppc64 ~x86"
 
+BDEPEND="virtual/pkgconfig"
 RDEPEND="media-libs/netpbm:="
-DEPEND="${RDEPEND}"
+DEPEND="
+	${RDEPEND}
+	dev-libs/glib:2
```

Comment:
You didn't check linking there, right?

Reply:
I guess so. No clue, why I've forgotten it. My fault :/

Reply:
ping ;)

Reply:
Jetzt :P

Reply:
I don't see it?